### ISSUE TITLE
Abnormal circulating Dipeptidyl Peptidase IV concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5995]
**Preferred term label:** Abnormal circulating Dipeptidyl Peptidase IV concentration

**Synonyms** Abnormal circulating DPP4; Abnormal circulating DPP IV

**Definition (free text, please give PubMed ID)** Concentration of dipeptidyl peptidase IV in the blood is outside the normal range., PMID: 29692250

**Parent term (use hpo.jax.org/app)** Abnormality of circulating enzyme level HP:0011021

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased circulating Dipeptidyl Peptidase IV concentration

**Synonyms** Decreased circulating DPP4; Decreased circulating DPP IV

**Definition (free text, please give PubMed ID)** Concentration of dipeptidyl peptidase IV in the blood is below the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Dipeptidyl Peptidase IV concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Increased circulating Dipeptidyl Peptidase IV concentration

**Synonyms** Increased circulating DPP4; Increased circulating DPP IV

**Definition (free text, please give PubMed ID)** Concentration of dipeptidyl peptidase IV in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Dipeptidyl Peptidase IV concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is not a clinical test that has a normal range such that an individual can display an "abnormal" value for it. A good way of deciding whether a lab test is a valid HPO term is to check whether there is a LOINC code for the corresponding test. If there is, that means that it is being tested clinically in at least some situations.
